doc/OnlineDocs/explanation/modeling/dae.rst

```diff
@@ -35,19 +35,19 @@ Pyomo.DAE introduces three new modeling components to Pyomo:
 
 As will be shown later, differential equations can be declared using
 using these new modeling components along with the standard Pyomo
-:py:class:`Var<.Var>` and
-:py:class:`Constraint<.Constraint>` components.
+:py:class:`.Var` and
+:py:class:`.Constraint` components.
 
 ContinuousSet
 *************
 
 This component is used to define continuous bounded domains (for example
 'spatial' or 'time' domains). It is similar to a Pyomo
-:py:class:`Set <.Set>`  component and can be used to index things
+:py:class:`.Set`  component and can be used to index things
 like variables and constraints. Any number of
 :py:class:`ContinuousSets <pyomo.dae.ContinuousSet>` can be used to index a
 component and components can be indexed by both
-:py:class:`Sets <.Set>` and
+:py:class:`.Set` and
```

Comment:
Could you revert this one back to :py:class:`Sets <.Set>`? I wanted to use the plural of the class name here.